design-proposals/mass_machine_type_transition.md

```diff
@@ -171,59 +225,66 @@ As both the minimum supported machine type version and the latest machine type v
+					* VM machine type is less than the minimum supported machine type
+					* VM machine type is greater than or equal to the minimum supported machine type
 					* VM machine type is equal to `q35`
 		* `NAMESPACE` is not specified
 			* `RESTART_NOW` is **true**
 				* Running
-					* VM machine type version less than the minimum supported version
-					* VM machine type version is greater than or equal to the minimum supported version
+					* VM machine type is less than the minimum supported machine type
+					* VM machine type is greater than or equal to the minimum supported machine type
 					* VM machine type is equal to `q35`
 				* Not running
-					* VM machine type version less than the minimum supported version
-					* VM machine type version is greater than or equal to the minimum supported version
+					* VM machine type is less than the minimum supported machine type
+					* VM machine type is greater than or equal to the minimum supported machine type
 					* VM machine type is equal to `q35`
 			* `RESTART_NOW` is **false**
 				* Running
-					* VM machine type version less than the minimum supported version
-					* VM machine type version is greater than or equal to the minimum supported version
+					* VM machine type is less than the minimum supported machine type
+					* VM machine type is greater than or equal to the minimum supported machine type
 					* VM machine type is equal to `q35`
 				* Not running
-					* VM machine type version less than the minimum supported version
-					* VM machine type version is greater than or equal to the minimum supported version
+					* VM machine type is less than the minimum supported machine type
+					* VM machine type is greater than or equal to the minimum supported machine type
 					* VM machine type is equal to `q35`
 
 # Implementation Phases
-
- - [ ] Create MMTT package that Kubernetes job will use as an image
- - [ ] Create Kubernetes job yaml to be invoked with MMTT subcommand
- - [ ] Insert package and job into kubevirt/pkg/virtctl and implement the subcommand (name TBD)
+## Initial Phase
+This will be a bare-bones design that will be functional for the user, but with limited features that focus on the specific use case of upgrading the machine types of VMs with outdated or deprecated machine types to the latest supported machine type. The user will have the ability to use the command with the flags to specify by namespace, label-selector,  or to restart any running VMs immediately. In this stage, the minimum supported machine type and the latest machine type will be determined and stored internally; the user will not be able to configure what machine types they would like to use at this time. Additionally, to monitor and manage/delete the job, the user can utilize the `kubectl` commands. In future phases, we will add subcommands that allow the user to monitor and manage the specific machine type transition job directly.
```

Comment:
nit: break into more lines so it's easier to comment about a specific sentence :)